src/components/bridge/ChainLabel/index.tsx

```diff
@@ -44,9 +50,29 @@ export const ChainLabel = ({ direction }: ChainLabelInterface) => {
       </ChainLabelContainer>
     )
   }
+
+  const TeleportationDestChainLabel = () => {
```

Comment:
```suggestion
  const teleportationDestChainLabel = () => {
```
We should name HOC, Component, Class as starting with capital letter .